src/v2/cookbook/debugging-in-vscode.md

```diff
@@ -81,7 +83,7 @@ Click on the Debugging icon in the Activity Bar to bring up the Debug view, then
 2.  Open your favorite terminal at the root folder and serve the app using Vue CLI:
 
   ```
-  npm start
+  npm run serve
```

Comment:
Just a minor comment: should we consider `yarn serve` as an alternative here?

Reply:
As much as I love yarn, I think we're using `npm` across the docs. Also, Vue CLI generated projects are not yarn specific (no `yarn.lock` for example).

Reply:
Ok, just ignore the comment above then 😅